modules/volunteers/admin/pages/pre.inc

```diff
@@ -44,8 +44,9 @@ if (!empty($_REQUEST)) {
         if (isset($_SESSION['IS_ADMIN']) || $_SESSION['IS_VOLUNTEERS']) {
             $dept = $updateData['dept_report'];
             $name = $updateData['dept_name'];
-            $report = volunteers_in_dept(intval($dept));
-            output_csv($report, "department_report_".$name.".csv");
+            $event = current_eventID();
```

Comment:
Redundant with the way you're already initializing this in `volunteers_in _dept`.

Also, what if I really **want** a list of vols across all years?

Reply:
Yes I am being redundant but part of me is waiting for the request to run reports or do actions on historical data. So as long as I keep things able to do that it makes it easier to fufill that potential future request.

Reply:
as for the multiple years that makes sense.  I will work it out to allow for that.

Reply:
Patch updated to allow for multiple years or all years. So when/if we add UI for that we can make it work.

Reply:
👍